Tests/SentryProfilerTests/SentryContinuousProfilerTests.swift

```diff
@@ -178,7 +178,7 @@ private extension SentryContinuousProfilerTests {
         let debugMeta = try XCTUnwrap(profile["debug_meta"] as? [String: Any])
         let images = try XCTUnwrap(debugMeta["images"] as? [[String: Any]])
         XCTAssertFalse(images.isEmpty)
-        let firstImage = images[0]
+        let firstImage = try XCTUnwrap(images.first)
```

Comment:
l: Does this change really matter? Both implementations will throw an error if there is no item in the list.

Reply:
If there's no item in the list, this results in an array out of bounds exception, and the test process stops. By throwing an error from a `throws` test case, the test is marked as failed, and the test process can go on to run subsequent test cases.